src/code42cli/cmds/securitydata.py

```diff
@@ -172,6 +168,12 @@ def clear_checkpoint(state, checkpoint_name):
     "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
 )
 @sdk_options()
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display complete list of data items or only from top level of nested response.",
```

Comment:
I don't think `or only` here makes sense either...  

How about just `Display complete list of data items from the first level of the nested response`

Reply:
same as [above comment](https://github.com/code42/code42cli/pull/133#discussion_r468312462)

Reply:
I would prefer @unparalleled-js 's proposed verbiage here.

Reply:
changed.